src/components/hostFamilyCard/HostFamilyCard.tsx

```diff
@@ -19,68 +19,55 @@ type Props = {
+      <Link to={`/family-details/${hostFamily.uid}`} state={hostFamily}>
+        <CardHeader className="w-full">
+          <img
+            className="w-full object-cover object-center"
+            src={hostFamily.photoUrl || defaultImage}
+            alt={hostFamily.displayName ?? "Family"}
+          />
+        </CardHeader>
 
-              <CardContent className="flex-grow">
-                  <CardContent>
-                      <div className="flex flex-col lg:flex-row sm:justify-between items-center">
-                          <div className="flex gap-2 mb-4 lg:mb-0 flex-col">
-                              <ItemInfo
-                                  nativeLanguage={
-                                      hostFamily.familyLangages ?? "Langues ?"
-                                  }
-                                  icon={<Flag />}
-                              />
-                              <ItemInfo
-                                  region={hostFamily.region}
-                                  icon={<MapPin />}
-                              />
-                              <ItemInfo
-                                  price={hostFamily.familyDalyRate ?? 0}
-                                  icon={<Euro />}
-                              >
-                                  / jour
-                              </ItemInfo>
-                          </div>
-                          <Button
-                              className="w-full lg:w-fit"
-                              onClick={() => navigate("/not-sale")}
-                          >
-                              Réserver
-                          </Button>
-                      </div>
-                  </CardContent>
-                  <CardContent className="border-t border-t-secondary pt-4">
-                      <CardTitle className="text-secondary text-balance py-2">
-                          Bienvenue chez la famille{" "}
-                          <span className="text-gray-600">
-                              {hostFamily.displayName}
-                          </span>
-                      </CardTitle>
+        <CardContent className="flex-grow">
+          <CardContent>
+            <div className="flex flex-col lg:flex-row sm:justify-between items-center">
+              <div className="flex gap-2 mb-4 lg:mb-0 flex-col">
+                <ItemInfo
+                  nativeLanguage={hostFamily.familyLangages ?? "Langues ?"}
+                  icon={<Flag />}
+                />
+                <ItemInfo region={hostFamily.region} icon={<MapPin />} />
+                <ItemInfo
+                  price={hostFamily.familyDalyRate ?? 0}
+                  icon={<Euro />}
+                >
+                  / jour
+                </ItemInfo>
+              </div>
+              <Button
+                className="w-full lg:w-fit"
+                onClick={() => navigate("/family-details/:id")}
```

Comment:
Attention quand on clique sur la card sauf sur le bouton reserver on renvoi sur `<Link to={`/family-details/${hostFamily.uid}`} state={hostFamily}>` [Voir ligne 26]
Quand on clique sur reserver on renvoi sur stripe ! ou alors il faut changer `Reserver` par `plus d'infos` par exemple.
En plus ton ` onClick={() => navigate("/family-details/:id")}` pas sur qu'il fonctionne correctement puisque le id est en dur